airflow/providers/databricks/hooks/databricks.py

```diff
@@ -114,6 +116,53 @@ def from_json(cls, data: str) -> RunState:
         return RunState(**json.loads(data))
 
 
+class ClusterState:
```

Comment:
I think we can move it to a new module, wdyt?

Reply:
I agree, then `RunState` class also would be moved too?

Reply:
Hello, @Lee-W I want to know that you also agree with moving `RunState`, `ClusterState` class into `airflow/providers/databricks/utils/states.py`? If so, I will move the classes into that file :)

Reply:
Either way works for me. 🙂

Reply:
OK, then I will not move and left them one it! :)